pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -1,12 +1,16 @@
 { lib, pkgs, stdenv, buildGoModule, fetchFromGitHub, nixosTests
-, nodejs, debianutils, mkdocs, python3, python3Packages }:
+, nodejs, debianutils, mkdocs, python3, python3Packages
+, pkg-config, pixman, cairo, pango }:
 
 
 let
   nodeDependencies = (import ./node-composition.nix {
     inherit pkgs nodejs;
     inherit (stdenv.hostPlatform) system;
-  }).nodeDependencies;
+  }).nodeDependencies.override {
+    nativeBuildInputs = [ pkg-config ];
+    buildInputs = [ pixman cairo pango ];
+  };
```

Comment:
Just curious, why were these necessary? It seems to be building without these inputs already on `nodejs_18`, unless I'm misunderstanding

This looks merge-ready to me otherwise though. I was just wondering about that first

Reply:
Actually it doesn't build without. With `nodejs_14` it built just fine, but with 16 and 18 this was necessary. If you can get it to build without this, that would obviously be better!

Reply:
Huh, that's weird... I see those same results (on `x86_64-linux` nixos-unstable) from `nix why-depends`. Still, simply changing the node2nix stuff to `nodejs_18` results in build errors that can only be fixed by including the changes in `default.nix`. Perhaps because it's a sub-derivation (`node-dependencies-ntfy`) that gets built? But that also depends on `nodejs_18` in the end... I have no clue... :(

Reply:
Tbh I'm happy to merge as-is if it fixes it for you. It was just weird, because I did remove those lines from this branch to bw sure it still built fine, and it did (in fact it actually just substituted from the cache since without those lines this is a zero-rebuild PR)

Reply:
I'm completely baffled. I removed these lines and like you said it built just fine. On my other system I spent at least one or two hours to figure out how to make it build again, and now it just works... :S
(sorry for the commits spam, I rebased to master before testing again)